docs/installation/mac.md

```diff
@@ -84,7 +84,10 @@ tar -xzf elasticsearch-oss-7.5.2-darwin-x86_64.tar.gz
 cd elasticsearch-7.5.2/
```

Comment:
Maybe remove this portion because of the new more explicit step?

Reply:
That's fair! Just amended that commit per your great suggestion :)